modules/_labs/color-picker/react/stories/stories_ColorPicker.tsx

```diff
@@ -61,8 +61,8 @@ storiesOf('Labs|Color Picker/React', module)
     const [color, setColor] = React.useState(defaultColor);
     const [colorInputValidColor, setColorInputValidColor] = React.useState(defaultColor);
     const [colorInputValue, setColorInputValue] = React.useState(defaultColor);
-    const inputRef = React.useRef(null);
-    const popupRef = React.useRef(null);
+    const inputRef = React.useRef<HTMLInputElement>(null);
+    const popupRef = React.useRef<HTMLDivElement>(null);
```

Comment:
These could probably just be `Element`. I'm torn on whether we should try to keep refs 100% accurate unless we absolutely need to. It is harder to maintain